src/components/CheckList/CheckList.tsx

```diff
@@ -64,13 +81,28 @@ const CheckListContent = ({ onChangeViewType, viewType }: CheckListContentProps)
   const isAllSelected = selectedCheckIds.size === filteredChecks.length;
   const totalPages = Math.ceil(filteredChecks.length / CHECKS_PER_PAGE);
 
-  const handleFilterChange = (filters: CheckFiltersType) => {
-    setCheckFilters((cf) => ({
-      ...cf,
-      ...filters,
-    }));
+  const handleFilterChange = (filters: CheckFiltersType, type?: FilterType) => {
```

Comment:
I think type isn't optional here, right? If you didn't provide a `type` nothing would happen?

Reply:
You're right, it shouldn't be optional. Changed it 👍